### DISCUSSION TITLE
Where Do Muse Hub Downloads Go?

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [musescore/MuseScore#15658] [General]
So the AUR binary of Muse Hub is currently broken, so I tried a direct download using one of the URLS in the logs, which turned out to be a ".musedownload" file, which turned out to just be a torrent file in disguise. In fact, once I changed the extension to ".torrent," I was able to use it in qbittorrent and download the Muse Keys bundle...But NOW I can't figure out where that folder's supposed to go. I couldn't find any hints in the source code, and there definitely doesn't seem to be any mention of it in the developer's manual or forums...How do I install this???

**Comment by theofficialgman** [comment]
@neurodiverseEsoteric I was curious about this a well for a system that can't run muse-hub (arm64)

I installed muse-hub on my ubuntu x86_64 desktop and found that the application uses an install location of `/srv/muse-hub` by default
here is an example of the full path to the Dream Piono assets

`/srv/muse-hub/downloads/Instruments/Muse Keys/Dream Piano/Dream Piano.spx`
`/srv/muse-hub/downloads/Instruments/Muse Keys/Dream Piano/SFZ/Dream Piano.sts`
before you ask, yes everything in these folders is root owned

you will also need the proprietary bin file installed in your dynamic library path as well, otherwise the sounds aren't available even if you correctly populate those folders

muse-hub added it here on my ubuntu install `/usr/lib/libMuseSamplerCoreLib.so`. unforntunatly for me that means I can't load these sounds on arm64 (or anything other than x86_64) since this bin file isn't available

hope this helps. ping me if you need anything else
